### DISCUSSION TITLE
Is there any html template for trivy fs/k8s?

### DISCUSSION THREAD

**Discussion by cww1769** [aquasecurity/trivy#4075] [Q&A] [answered]
As the existing html.tpl is not suitable for trivy fs/k8s.

**Comment by knqyf263** [comment, marked as answer]
K8s scanning doesn't support templates. Filesystem scanning does.
https://aquasecurity.github.io/trivy/v0.41/docs/configuration/reporting/#template

**Reply by fatihtokus** [reply]
Hi @knqyf263, 

Is there a reason for that or is it just a lack of feature?

Regards,

**Reply by knqyf263** [reply]
It's just a lack.

**Reply by itaysk** [reply]
we want to keep trivy minimal and let all output options be implemented in separate projects (meaning trivy output json and you can use any tool to process it). in the future we have a plan to make this more streamlined with plugins

**Reply by fatihtokus** [reply]
Makes sense, as long as the JSON output includes all results.

**Comment by fatihtokus** [comment]
Hi @cww1769 

I have a trivy plugin(https://github.com/fatihtokus/scan2html) that creates a well-formated filterable/sortable html report. Please have a look and if you like it I can extend it to be suitable for fs/k8s as well.

Regards!

**Reply by cww1769** [reply]
Thanks so much, fatihokus. I am trying your project but I met some problems. I will create a issue

**Reply by fatihtokus** [reply]
Thanks for the feedback, I will be working on it as a priority, will update you soon!

**Reply by fatihtokus** [reply]
Hi @cww1769 ,

I have fixed the issue in the latest version v0.2.1. Could you please retry it with the latest version?

1. Install/update $ trivy plugin install github.com/fatihtokus/scan2html
2. $ trivy scan2html image proxy:v2.4.0 result.html

Thanks for using the plugin

**Reply by cww1769** [reply]
THanks a lot! it works now
